### PR TITLE
Event tokens clean up

### PR DESCRIPTION
**Pull request by larssandergreen**
Overview
----------------------------------------
Just a little clean up now that these are handled in a less adhoc way, following on #27333

### CODE REVIEW

File: CRM/Event/Tokens.php [hunk 1]
```diff
@@ -191,20 +191,15 @@ protected function getEventTokenValues(int $eventID = NULL): array {
...
           'is_public:label',
           'is_share',
           'is_share:label',
           'requires_approval',
           'requires_approval:label',
           'is_monetary:label',
-          'event_type_id:label',
           'event_type_id:name',
           'pay_later_text',
-          'pay_later_receipt',
```
Comment (larssandergreen) [inline CRM/Event/Tokens.php:207]: These are included in exposed fields below.

File: CRM/Event/Tokens.php [hunk 2]
```diff
@@ -215,16 +210,12 @@ protected function getEventTokenValues(int $eventID = NULL): array {
         'state_province' => $event['loc_block_id.address_id.state_province_id:label'],
         'postal_code' => $event['loc_block_id.address_id.postal_code'],
       ]);
-      $tokens['location']['text/html'] = nl2br(trim($tokens['location']['text/plain']));
```
Comment (larssandergreen) [inline CRM/Event/Tokens.php:218]: Theoretically, we've lost a trim() for the address block, but doesn't seem to be needed in testing (note that the token is not currently in use as it is not fully implemented (supplemental addresses, etc))

**Comment by eileenmcnaughton** [comment]
Jenkins not impressed

Note the event tokens are 'special' because they cache by entity - which is for performance reasons (real or perceived)

[Test Result](https://test.civicrm.org/job/CiviCRM-Core-Matrix-PR/4541/BKPROF=dfl,SUITES=phpunit-crm,label=bknix-tmp/testReport/) (2 failures / -7)
[CRM_Utils_TokenConsistencyTest.testEventTokenConsistency](https://test.civicrm.org/job/CiviCRM-Core-Matrix-PR/4541/BKPROF=dfl,SUITES=phpunit-crm,label=bknix-tmp/testReport/junit/(root)/CRM_Utils_TokenConsistencyTest/testEventTokenConsistency/)
[CRM_Utils_TokenConsistencyTest.testEventTokenConsistencyNoParticipantTokens](https://test.civicrm.org/job/CiviCRM-Core-Matrix-PR/4541/BKPROF=dfl,SUITES=phpunit-crm,label=bknix-tmp/testReport/junit/(root)/CRM_Utils_TokenConsistencyTest/testEventTokenConsistencyNoParticipantTokens/)

**Comment by larssandergreen** [comment]
@eileenmcnaughton It's an extra newline at the end of the address. How would you feel about just rtrimming fields in general when converting from text to html? Seems like this probably will come up more than once.

**Comment by eileenmcnaughton** [comment]
@larssandergreen seems OK to rtrim

**Comment by larssandergreen** [comment]
@eileenmcnaughton Added the rtrim newlines only in here as well.

Yes, I saw that the event tokens path was very odd with the caching. Confusing!

**Pull request merged**